x/oracle/types/keys.go

```diff
@@ -18,5 +20,26 @@ const (
 )
 
 var (
-// KeyPrefixOracles defines key to store oracle
+	// KeyPrefixOracle defines key to store oracle
+	KeyPrefixOracle                 = []byte{0x01}
+	KeyPrefixOracleRegistration     = []byte{0x02}
+	KeyPrefixOracleRegistrationVote = []byte{0x03}
+
+	KeyIndexSeparator = []byte{0xFF}
 )
+
+func GetKeyPrefixOracle(address string) []byte {
+	return append(KeyPrefixOracle, []byte(address)...)
+}
+
+func GetKeyPrefixOracleRegistration(address string) []byte {
+	return append(KeyPrefixOracleRegistration, []byte(address)...)
+}
+
+func GetKeyPrefixOracleRegistrationVote(uniqueID, votingTargetAddress, voterAddress string) []byte {
```

Comment:
May I ask why the unique ID is needed to store the OracleRegistrationVote?

Reply:
Because we have to consider OracleRegistration and OracleUpgrade.
It will probably be stored in this KV store based on uniqueID during OracleUpgrade.

Reply:
Ah I missed that point. Ok then, it makes sense to me.